README.md

```diff
@@ -3,30 +3,30 @@
 
 <p align="left" >
 <img src="https://s-media-cache-ak0.pinimg.com/736x/aa/bc/3b/aabc3b2b789f478ffb87ac2f0bdd2d33--ergo-proxy-manga-anime.jpg" width="250" align="center" />
-<span>Ergo Proxy - The reverse proxy agent for local domain management.</span>
+<span>Ergo Proxy - Reverse proxy agent for local domain.</span>
```

Comment:
I am not sure about this. it sounds strange to me. I would rather stick with the previous version cause it explains better the app purpose. What do you think?